plugins/org.komodo.relational/src/org/komodo/relational/internal/RelationalChildRestrictedObject.java

```diff
@@ -33,6 +34,16 @@ protected RelationalChildRestrictedObject( final UnitOfWork uow,
     /**
      * {@inheritDoc}
      *
+     * @see org.komodo.repository.ObjectImpl#getChildTypes()
+     */
+    @Override
+    public final KomodoType[] getChildTypes() {
+        return KomodoType.NO_TYPES;
+    }
```

Comment:
Seems odd too given this is a RelationalChildRestrictedObject. Unless, the name implies it has no children?